conda/common/io.py

```diff
@@ -486,11 +536,32 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         if self.start_time:
+            entry_name = self.entry_name
             end_time = time()
             run_time = end_time - self.start_time
+            self.total_call_num[entry_name] += 1
+            self.total_run_time[entry_name] += run_time
             self._ensure_dir()
             with open(self.record_file, 'a') as fh:
-                fh.write("%s,%s\n" % (self.entry_name, run_time))
+                fh.write("%s,%f\n" % (entry_name, run_time))
+            # total_call_num = self.total_call_num[entry_name]
+            # total_run_time = self.total_run_time[entry_name]
+            # log.debug(
+            #     '%s %9.3f %9.3f %d', entry_name, run_time, total_run_time, total_call_num)
```

Comment:
I essentially just added `total_run_time, total_call_num` to be able to get a "live" output of accumulated numbers via that log statement. So that was just a convenience thing during benchmarking for me..